src/Reducers/reducer.js

```diff
@@ -0,0 +1,51 @@
+import { ADD_TODO, CHANGE_STATUS } from "../Constant/index";
+
+const initialTodo = {
+  todos: [
+    
+  ],
+};
+
+const addTodoReducer = (state = initialTodo, action) => {
+  switch (action.type) {
+    case ADD_TODO: {
+      if (action.text.trim().length > 0) {
+        return {
+          todos: [
+            ...state.todos,
+            {
+              text: action.text.trim(),
+              key: Math.random().toString(),
+              check: false,
+            },
+          ],
+        };
+      } else {
+        alert("List can't be empty", [
+          { text: "OK", onPress: () => console.log("Alert Closed") },
```

Comment:
no need of this line.
as you are just consoling something.
just do this - 
` alert("List can't be empty")`